CHANGELOG.rst

```diff
@@ -1,5 +1,61 @@
 Changelog
 =========
+1.13 (2020-12-22)
+-----------------
+MLflow 1.13 includes several major features and improvements:
+
+Features:
+
+New fluent APIs for logging in-memory objects as artifacts:
+
+- Add ``mlflow.log_text`` which logs text as an artifact (#3678, @harupy)
+- Add ``mlflow.log_dict`` which logs a dictionary as an artifact (#3685, @harupy)
+- Add ``mlflow.log_figure`` which logs a figure object as an artifact (#3707, @harupy)
+- Add ``mlflow.log_image`` which logs an image object as an artifact (#3728, @harupy)
+
+UI update / fixes (#3867, @smurching):
+
+- Add "Empty state" overlay for experiments
```

Comment:
This is a small change. Let's move it down to the "Small changes, bug fixes, and doc updates" section (see https://github.com/mlflow/mlflow/pull/3890/files#r548237542 for context)

Reply:
Done!